app/src/main/java/io/github/muntashirakon/setedit/SetActivity.java

```diff
@@ -0,0 +1,63 @@
+package io.github.muntashirakon.setedit;
+
+import android.app.Activity;
+import android.content.ComponentName;
+import android.content.Context;
+import android.os.Bundle;
+
+import io.github.muntashirakon.setedit.Utils.EditorUtils;
+import io.github.muntashirakon.setedit.adapters.SettingsRecyclerAdapter;
+
+public class SetActivity extends Activity {
+    private String settingsType;
+    private String keyName;
+    private String KeyValue;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Bundle bundle = getIntent().getExtras();
+        int i = 0;
+        while (bundle.getString("settingsType" + i) != null &&
+                !bundle.getString("settingsType" + i).isEmpty()) {
+            settingsType = bundle.getString("settingsType" + i);
+            keyName = bundle.getString("MyKeyName" + i);
+            if (bundle.getBoolean("delete" + i)) {
+                SetActivityShortcutDelete();
+            } else {
+                KeyValue = bundle.getString("KeyValue" + i);
+                SetActivityShortcutUpdate();
+            }
+            i++;
+        }
+    }
+
+    public ComponentName SetActivityShortcut() {
```

Comment:
A method name begins with a small letter, not capital letter.

Reply:
Corrected, and I'm sorry about this beginner's error.